src/include/duckdb/planner/constraints/bound_foreign_key_constraint.hpp

```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// duckdb/planner/constraints/bound_foreign_key_constraint.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "duckdb/common/unordered_set.hpp"
+#include "duckdb/planner/bound_constraint.hpp"
+
+namespace duckdb {
+
+class BoundForeignKeyConstraint : public BoundConstraint {
+public:
+	BoundForeignKeyConstraint(ForeignKeyInfo info, unordered_set<idx_t> pk_key_set, unordered_set<idx_t> fk_key_set)
+	    : BoundConstraint(ConstraintType::FOREIGN_KEY), info(move(info)), pk_key_set(move(pk_key_set)),
+	      fk_key_set(move(fk_key_set)) {
+#ifdef DEBUG
+		D_ASSERT(info.pk_keys.size() == pk_key_set.size());
```

Comment:
These assertions are not testing anything useful, since they refer to the parameters `pk_key_set` which are moved in the constructor. The sets will always both be empty.

I recommend renaming the parameters to `info_p`, `pk_key_set_p` and `fk_key_set_p` to avoid this problem.

Reply:
Your are right, I will fix it